src/libhirte/common/cfg.c

```diff
@@ -91,6 +96,32 @@ void cfg_dispose(struct config *config) {
         free(config);
 }
 
+bool cfg_set_port(enum NodeType node, void *instance, const char *port_s) {
```

Comment:
Based on the naming `cfg_set_port` it suggests that this function sets the port on the config, but it actually sets it on the passed in agent/manager instance.

Reply:
I thought the same, we could change that too but looks like we will drop this patch.